### DISCUSSION TITLE
Is it possible to have product with unit?

### DISCUSSION THREAD

**Discussion by ivsaneesh** [vendurehq/vendure#1574] [Q&A]
Hi,
I am trying to create an ecommerce site.
My site need to sell item with unit like product should have Kilogram, gram and all kinds of measurements.
Is there any option for this.

**Comment by michaelbromley** [comment]
Hi,

You can use [custom fields](https://www.vendure.io/docs/developer-guide/customizing-models/#customizing-models-with-custom-fields) to add this kind of additional data to your product variants. Do you need to be able to track stock levels of arbitrary fractional units? E.g. do you want to be able to know that you have 10kg of potatos in stock, you sold 2.5kg, so you have 7.5kg left? Or is that not a requirement?
